component-library/component-lib/src/lib/shared/breadcrumb/breadcrumb.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, Input, OnInit, OnChanges, SimpleChanges } from '@angular/core';
+import { DSSizes } from "../../../shared/constants/jl-components/jl-components.constants/jl-components.constants";
+import { ILinkComponentConfig } from "./link/link.component";
+import { TranslateService } from "@ngx-translate/core";
+import { StandAloneFunctions } from "../../../shared/functions/stand-alone.functions";
+
+export enum LinkType {
+  href = 'href',
+  routerLink = 'routerLink',
+}
+
+export interface IBreadcrumbConfig {
+  //TODO: Make an enum for type (i.e. HREF, RouterLink, anchor)
```

Comment:
this is already done right? as i see enum with href and type, so can you remove this comment?